packages/ifpack2/test/unit_tests/Ifpack2_UnitTestAdditiveSchwarz.cpp

```diff
@@ -1249,6 +1249,300 @@ TEUCHOS_UNIT_TEST_TEMPLATE_3_DECL(Ifpack2AdditiveSchwarz, SGS_Overlap, Scalar, L
   TEST_COMPARE_FLOATING_ARRAYS(sol->get1dView(), x->get1dView(), 4*STS::eps ());
 }
 
+TEUCHOS_UNIT_TEST_TEMPLATE_3_DECL(Ifpack2AdditiveSchwarz, FastILU, Scalar, LocalOrdinal, GlobalOrdinal)
```

Comment:
Put the `#ifdef` on the outside, since you don't want to give users the impression that the test ran, if it didn't do any work.